gpiozero/boards.py

```diff
@@ -1260,11 +1263,19 @@ def decimal_point(self):
         if len(self) > 7:
             return self[7].value 
         else:
-            raise OutputDeviceError('There is no 8th pin for the decimal point')
+            raise OutputDeviceError('there is no 8th pin for the decimal point')
     
     @decimal_point.setter
     def decimal_point(self, value):
         if len(self) > 7:
             self[7].value = value
         else:
-            raise OutputDeviceError('There is no 8th pin for the decimal point')
\ No newline at end of file
+            raise OutputDeviceError('there is no 8th pin for the decimal point')
+            
+    def set_char_layout(self, char, layout):
```

Comment:
1. A setter method is pythonic?
2. For me the `set_char_layout` gives the impression that I **can only change** an existing layout.
2. If the layout is attribute (`self._layout`) not a static method, if I change a char layout of a 7seg display, then is desirable that another 7seg haven't this layout?